docs/aurora/workflows/libensemble.md

```diff
@@ -1 +1,108 @@
 # libEnsemble on Aurora
+
+[libEnsemble](https://libensemble.readthedocs.io/en/main/) is a Python toolkit for running dynamic ensembles of calculations.
+
+Users provide generator and simulator functions to express their ensembles, where the generator can steer the ensemble based on previous results. These functions can portably submit user executables at any scale.
+
+System details are detected, and dynamic resource management is provided. This includes automatically detecting, assigning, and reassigning GPUs for ensemble members.
+
+libEnsemble can be used in a consistent manner on laptops, clusters, and supercomputers with minimal required dependencies.
+
+## Configuring Python and Installation
+
+To obtain Python use:
+
+    module use /soft/modulefiles
```

Comment:
no need for this on Aurora, right now. Currently it is available in the default `MODULEPATH`, specifically:
```
/opt/aurora/24.180.3/modulefiles
```

admittedly, we arent entirely consistent with it, see: https://docs.alcf.anl.gov/aurora/data-science/frameworks/tensorflow/

so fine to leave as-is; it does no harm